BrainPortal/app/views/signups/new.html.erb

```diff
@@ -80,7 +80,7 @@
     <tr>
       <th><%= f.label(:position, "* Current position or role:") %></th>
       <td><%= f.select :position, options_for_select(
-              [ [ "(select one)", "" ], "Faculty", "PostDoc", "Researcher", "Student", "Industry", "Other" ],
+              [ [ "(select one)", "" ], "Undergraduate Student", "Postgraduate Student", "Post Doctoral Student", "Faculty", "Researcher", "Staff", "Industry", "Other" ],
```

Comment:
I like the new names, but why change the order? Put Faculty first and then go down the list of academic levels (including just "student", not everybody is at a university or college). I'll come up wityh a better list in a follow up comment.